docs/wiki/The CI environment - Pipeline design.md

```diff
@@ -121,3 +122,46 @@ Once triggered, the pipeline crawls through the library and updates the tables i
 The purpose of the wiki pipeline is to sync any files from the `docs/wiki` folder to the wiki repository. It is triggered each time changes are pushed to the `main` branch and only if files in the `docs/wiki` folder are altered.
 
 > **Note:** Any changes performed directly on the wiki via the UI will be overwritten by this pipeline.
+
+## PSRule Pre-Flight validation pipeline
+
+The purpose of the PSRule Pre-Flight validation pipeline is to validate Azure resources deployed by module validation pipeline tests, by leveraging [PSRule for Azure](https://azure.github.io/PSRule.Rules.Azure/about/).
+PSRule for Azure is aligned to the [Azure Well-Architected Framework (WAF)](https://learn.microsoft.com/en-us/azure/architecture/framework/). Tests called rules check the configuration of Azure resources against WAF principles.
+
+The pipeline, currently only available as a [GitHub workflow](https://github.com/Azure/ResourceModules/blob/main/.github/workflows/platform.librarycheck.psrule.yml), runs weekly on the whole library, providing as output the list of non compliant resources and corresponding failing rules, if any.
+
+### Configuration settings
+
+PSRule options set for the CARML repository are configured in the [ps-rule.yaml](https://github.com/Azure/ResourceModules/blob/main/ps-rule.yaml) file.
+
+Documentation for all configuration options is available at the following links:
+- https://aka.ms/ps-rule/options
+- https://aka.ms/ps-rule-azure/options
+
+### Baselines
+
+A [baseline](https://azure.github.io/PSRule.Rules.Azure/working-with-baselines/) is a standard PSRule artifact that combines rules and configuration. This pipeline uses the default baseline to analyze module test resources.
```

Comment:
Which pipeline?

Reply:
The one described by the main section 😀. If not clear enough I can repeat

Reply:
Would be great to call it out by name, yes. When I was reading the paragraph I got confused because I thought there is something like a pipeline specifially for baselines, or something like that.